olwebsite-app/src/pages/NotFoundView.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const NotFoundView = () => {
```

Comment:
Should make an issue to improve the NotFoundView. Should be an error well with Page Not Found message

Also "Page Not Found" sounds better

Reply:
I'll do that. I wasn't sure what was going to be on it, so I stubbed it to be revisited once we got fleshed out.

Reply:
I made an issue for it already, sorry i didnt say anything

Reply:
Ok that's fine.